src/Pretzel.Logic/Templating/Context/SiteContextGenerator.cs

```diff
@@ -186,12 +186,12 @@ private bool ContainsYamlFrontMatter(string file)
             return postFirstLine != null && postFirstLine.StartsWith("---");
         }
 
-        private bool IsExcludedPath(string relativePath)
+        public bool IsExcludedPath(string relativePath)
         {
             return excludes.Contains(relativePath) || excludes.Any(e => relativePath.StartsWith(e));
         }
 
-        private bool IsIncludedPath(string relativePath)
+        public bool IsIncludedPath(string relativePath)
```

Comment:
True, I only changed visibility to be "semantically symmetric" with IsExcludedPath